histogram.js

```diff
@@ -72,27 +75,41 @@ class HistogramElement extends HTMLElement {
     this._histo = core({
       title: this.label,
     });
-    
     this.shadowRoot.appendChild(this._histo.el);
-
     const broker = getDataBroker(this);
+
+    function toggleSVGAndLoader(svgDisplay, loaderDisplay) {
+      const loader = this.shadowRoot.querySelector('.samplingLoader');
+      const svgElements = this.shadowRoot.querySelectorAll('.iobio-container');
```

Comment:
This is relying on the implementation of the `iobio.viz` library. For example, if iobio.viz changes to use a name other than "iobio-container", our code will break. We should only interact with elements that we control. In this case, I think `iobio-histogram-svg-container` is what you want.

Reply:
Yeah, I thought about that case and tried to select` iobio-histogram-svg-container ` before. But I got the error below sending back from iobio.viz. Any thoughts about this error?
<img width="464" alt="Screenshot 2024-07-30 at 3 23 22 PM" src="https://github.com/user-attachments/assets/a23639fd-a4ab-4b1c-886a-947b2974b590">

Reply:
I suspect this is happening because the element is completely removed from layout if you set `display: none`. Try using `visibility: hidden` and see if that works. See here: https://stackoverflow.com/a/19400181/943814